zeppelin-zengine/src/main/java/org/apache/zeppelin/helium/Helium.java

```diff
@@ -411,7 +412,7 @@ public HeliumPackageSuggestion suggestApp(Paragraph paragraph) {
    * @return ordered list of enabled buildBundle package
    */
   public List<HeliumPackage> getBundlePackagesToBundle() {
-    Map<String, List<HeliumPackageSearchResult>> allPackages = getAllPackageInfoWithoutRefresh();
+    Map<String, List<HeliumPackageSearchResult>> allPackages = getAllPackageInfo();
```

Comment:
I did change in a few places withoutRefresh() to the default/refresh option so the REST APIs could run without error independently. In this case, and from my tests, it's safe to not refresh as this is called in the constructor. I applied the rollback on latest commit.